features/step_definitions/json_api_average.rb

```diff
@@ -9,9 +9,9 @@
 end
 
 Given(/^a set of ratings for several songs$/) do
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 1 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 2 }
-  6.times { |i| create :song_rating, value: i + 1, ratable_id: 3 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 1 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 2 }
+  6.times { |i| create :song_rating, value: (i + 1) - 1, ratable_id: 3 }
```

Comment:
The first time through the value is set to 0 because (0 +1) - 1 is indeed 0. However, i was incremented up by 1 in the next loop, so 1 + 1 - 1 = 1. Basically it's just a cheap way to offset the entire range of i through the series of loops by 1. Instead of 1 - 6, we get 0 - 5.